sentinelone_connector.py

```diff
@@ -179,7 +179,7 @@ def _handle_test_connectivity(self, param):
                 }
         }
         ret_val, response = self._make_rest_call(
-            '/web/api/v2.1/users/login/by-api-token', action_result, params=None, headers=header, data=json.dumps(body), method='post')
+            '/web/api/v2.1/accounts', action_result, params=None, headers=header, data=json.dumps(body), method='get')
```

Comment:
I just wanted to confirm, you are doing a GET with no query parameters but passing JSON as the body? Shouldn't this be a POST?

Reply:
correct.  api-token api is not allowed for service account. 
this is what customer returned to us after updating the app (2.2.7 is the previously released version, 2.2.8 is at customer location)

diff -r -N -u 227/sentinelone/sentinelone_connector.py 228/sentinelone/sentinelone_connector.py
--- 227/sentinelone/sentinelone_connector.py	2023-01-24 14:21:30.000000000 -0800
+++ 228/sentinelone/sentinelone_connector.py	2023-01-30 18:47:31.000000000 -0800
@@ -179,7 +179,7 @@
                 }
         }
         ret_val, response = self._make_rest_call(
-            '/web/api/v2.1/users/login/by-api-token', action_result, params=None, headers=header, data=json.dumps(body), method='post')
+            '/web/api/v2.1/accounts', action_result, params=None, headers=header, data=json.dumps(body), method='get')
         if phantom.is_fail(ret_val):
             self.save_progress("Test Connectivity Failed")
             return action_result.get_status()

Reply:
OK, I am a bit surprised by this but I suppose if it works?

Reply:
Yes. That's what I was told. so we're incorporating the same change as we don't have an asset to verify